### PR TITLE
Fix: Inner blocks control position for Cart Block

### LINKED ISSUE #6698: Cart Block: Order summary inner blocks controls appear on the left side of the screen
## Describe the bug

Cart Block: Order summary inner blocks controls appear on the left side of the screen

## To reproduce

Steps to reproduce the behaviour:

1. Go to the Cart Block
2. Click on the fee inner block
3. Notice that the controls appear on the left side

## Expected behaviour

The controls should appear aligned with the left margin of the Order summary block. Like in the Checkout block.

## Screenshots
Screenshot 2022-07-18 at 17 45 35

### PR DESCRIPTION
**Pull request by tarunvijwani**
## Describe the bug

Cart Block: Order summary inner blocks controls were appearing on the left side of the screen

## Changes in the PR

Removed `__experimentalExposeControlsToChildren` from `cart/index.js` and `cart/edit.js` file. 

`__experimentalExposeControlsToChildren`  was added by the [PR](https://github.com/woocommerce/woocommerce-blocks/pull/4817) to display the view switcher when selecting the inner blocks of the Cart block. It was dependent on [PR opened in Gutenberg repo](https://github.com/WordPress/gutenberg/pull/35078), which was never merged. 

Currently, the view switcher is only visible when we select the parent Cart block, which is nearly impossible to select without using the list view option. It can be done now using a filter, I'll create a different ticket to track the issue 

Fixes #6698

### Screenshots

| Before | After |
| ------ | ----- |
|Screenshot 2022-07-18 at 17 45 35|image|

#### Automated Tests
* [x] Changes in this PR are covered by Automated Tests.
  * [x] Unit tests
  * [ ] E2E tests

#### User Facing Testing

1. Go to the Cart Block
2. Click on the fee inner block
3. Confirm that the controls appear on the correct side

* [ ] Do not include in the Testing Notes 

### WooCommerce Visibility

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Changelog

> Fix: Inner blocks control position for Cart Block

### CODE REVIEW

File: assets/js/blocks/cart/edit.js [hunk 1]
```diff
@@ -101,9 +101,12 @@ export const Edit = ( { className, attributes, setAttributes, clientId } ) => {
 					currentView={ currentView }
 					previewData={ { previewCart } }
 				>
-					<BlockControls __experimentalShareWithChildBlocks>
-						{ ViewSwitcherComponent }
-					</BlockControls>
+					<BlockSettings
+						attributes={ attributes }
+						setAttributes={ setAttributes }
+					/>
```
Comment (tarhi-saad) [inline assets/js/blocks/cart/edit.js]: Hi @tarunvijwani! The `BlockSettings` component is already added on line 80 of this file. With the current two `BlockSettings` we have double dark mode controls:

image

**Review by tarhi-saad — CHANGES_REQUESTED** [review]
Hi @tarunvijwani! Thank you for fixing the block's control position bug! 🙌 You can find below my comment about a display bug!

File: assets/js/blocks/cart/edit.js (hunk 1 above)
Comment (tarunvijwani) [reply assets/js/blocks/cart/edit.js]: Sorry, I missed them while resolving the merge conflict. I have removed the duplicate code. Thank you!

**Review by tarhi-saad — APPROVED** [review]
That's perfect, @tarunvijwani! It looks good to me! So, let's 🚀 it! 💪

**Pull request merged**